### PR TITLE
feat: k8s deploy example

### PR DESCRIPTION
**Pull request by numb3r3**
**Note**:

- as an example, only the `indexer-0.yml` works with PVC.

references:
- https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/

### CODE REVIEW

File: examples/k8s_example/deployment/indexer/indexer-2.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        pea_type: WORKER
+        shard_id: '2'
+    spec:
+      containers:
+      - args:
+        - executor
+        - --name
+        - indexer-2
+        - --workspace
+        - /Users/fengwang/Jina/workspace/pqlite/examples/k8s_example
```
Comment (numb3r3) [inline examples/k8s_example/deployment/indexer/indexer-2.yml:91]: The workspace folder should be changed to container path

File: examples/k8s_example/deployment/indexer/indexer-1.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        pea_type: WORKER
+        shard_id: '1'
+    spec:
+      containers:
+      - args:
+        - executor
+        - --name
+        - indexer-1
+        - --workspace
+        - /Users/fengwang/Jina/workspace/pqlite/examples/k8s_example
```
Comment (numb3r3) [inline examples/k8s_example/deployment/indexer/indexer-1.yml:91]: This path configuration would not work

**Review by JoanFM — COMMENTED** [review]
But do u need to do any change besides that? for the Persistent volume?

**Comment by numb3r3** [comment]
I need to add the PVC in the generated YAML file (e.g., `indexer-0.yml`)

```
    volumeMounts:
          - mountPath: "/workspace/workspace"
            name: indexer-pv-storage # this PVC is created from `pv_claim.yml`
```

**Comment by JoanFM** [comment]
> I need to add the PVC in the generated YAML file, as shown in`indexer-0.yml`
> 
> ```
>     volumeMounts:
>           - mountPath: "/workspace/workspace"
>             name: indexer-pv-storage # this PVC is created from `pv_claim.yml`
> ```

Then focus on the changes in README, not just on what is dumped

**Comment by numb3r3** [comment]
Yes, I will add the guidelines into README. I will continue this example after this https://github.com/jina-ai/jina/pull/4160 is closed.

**Comment by numb3r3** [comment]
close it due to redundant with https://github.com/jina-ai/pqlite/pull/97

**Pull request closed without merging**